welcome/index.adoc

```diff
@@ -277,29 +283,36 @@ You can xref:../storage/expanding-persistent-volumes.adoc#expanding-persistent-v
 
 ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 - **xref:../applications/quotas/quotas-setting-per-project.adoc#quotas-setting-per-project[Set resource quotas]**: Choose from CPU, memory, and other system resources to xref:../applications/quotas/quotas-setting-per-project.adoc#quotas-setting-per-project[set quotas].
-endif::[]
+endif::openshift-enterprise,openshift-webscale,openshift-origin[]
 
 - **xref:../applications/pruning-objects.adoc#pruning-objects[Prune and reclaim resources]**: Reclaim space by pruning unneeded Operators, groups, deployments, builds, images, registries, and cron jobs.
 
 - **xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-infrastructure-practices.adoc#scaling-cluster-monitoring-operator[Scale] and xref:../scalability_and_performance/using-node-tuning-operator.adoc#using-node-tuning-operator[tune] clusters**: Set cluster limits, tune nodes, scale cluster monitoring, and optimize networking, storage, and routes for your environment.
 
-- **Update a cluster**:
+- **xref:..//updating/understanding_updates/intro-to-updates.html[Update a cluster]**:
```

Comment:
This is working as-is, but idk if the portal is pickier so best to correct it :sweat_smile: 
```suggestion
- **xref:../updating/understanding_updates/intro-to-updates.html[Update a cluster]**:
```

Reply:
:heavy_check_mark: